cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/auth/OpenStackClient.java

```diff
@@ -44,10 +44,29 @@ public KeystoneCredentialView createKeystoneCredential(AuthenticatedContext auth
 
     public Access createAccess(AuthenticatedContext authenticatedContext) {
         KeystoneCredentialView osCredential = createKeystoneCredential(authenticatedContext);
-        return OSFactory.builder().endpoint(osCredential.getEndpoint())
-                .credentials(osCredential.getUserName(), osCredential.getPassword())
-                .tenantName(osCredential.getTenantName())
-                .authenticate().getAccess();
+        if ( osCredential.getVersion().equals(KeystoneCredentialView.CB_KEYSTONE_V2) )
```

Comment:
Not sure why IDEA did not report this even though I had the config.jar imported. Have fixed all the checkstyle issues - awaiting your response on the other comment I posted for the validation check before I submit the pull request.